py3status/modules/net_rate.py

```diff
@@ -50,39 +65,36 @@ class Py3status:
     devfile = '/proc/net/dev'
     format = "{interface}: {total}"
     format_no_connection = ''
+    format_value = "{value:5.1f} {unit:>5s}"
```

Comment:
I think we should get one of these PRs done first because once we get one done the others should all be easy.  but yes I think it'd be nicer if nothing else we don't get the 2 spaces issue for B/s which I think looks a little nasty